script/post_generate_language

```diff
@@ -4,3 +4,18 @@ cd "`dirname \"$0\"`/.."
 
 echo "Removing trailing whitespace..."
 find lib -name "*.rb" -type f -exec sed -E -i '' 's/[[:space:]]+$//g' {} +
+
+echo "Removing Apache License Annotations..."
+find lib -name "*.rb" -type f -exec ruby script/remove_apache_license.rb {} +
+ruby script/remove_apache_license.rb docraptor.gemspec
```

Comment:
I left the `spec` files alone, but they could be added. They were new and vaguely useful so I didn't delete them outright. Want to add those, too?